scripts/dashboard-importer/README.md

```diff
@@ -0,0 +1,185 @@
+
+#### Behavior and Output
+This command converts the specified JSON files and writes the converted files
+to the `reports/<date>/<time>/` directory of the importer. The directory also includes a `report.json` file. `report.json` logs details of the invocation
+along with warnings or errors from the conversion.
+
+### Upload Only
+To manually upload the converted dashboards, use the following command:
+```
+./upload.sh <FILE_OR_DIRECTORY> <PROJECT_ID>
+```
+#### Arguments:
+  - FILE_OR_DIRECTORY: Path to a file or directory that contains dashboards in Cloud Monitoring JSON format.
+  - PROJECT_ID (optional): The destination Google Cloud project for the converted dashboards. Defaults to the project set in your active gcloud config.
+
+#### Behavior and Output
+When uploading directories, the command prompts you to confirm the project and files before it uploads.
+
+After the upload is complete, the command prints out the URLs of the uploaded dashboards.
+In the directory with the file(s) to be uploaded, it also creates `upload_<time>.txt` file that records the names and URLs of the uploaded files
+
+## Troubleshooting
+Common troubleshooting cases for the Cloud Monitoring Dashboard Importer
+
+### gcloud Related Issues
+
+#### gcloud: command not found
+
+You need to install gcloud. See the
+[gcloud Installation Docs](https://cloud.google.com/sdk/docs/install)
+
+#### Problem refreshing current auth tokens
+
+You need to run `gcloud auth login`. See the [docs](https://cloud.google.com/sdk/gcloud/reference/auth/login) on obtaining access credentials.
+
+### Can't find output files
+
+Converted JSON files are written to the `reports/<date>/<time>/` directory
+of the importer.
+
+In the same directory, you will also find:
+
+-   `report.json` - contains information regarding the conversion such warnings
+    and errors that occurred
+-   `upload_<time>.txt` - record of the generated cloud monitoring urls from uploading
+
+### Conversion Issues
+
+#### Dashboard has tiles with no data coming in
+
+We recommend the following steps for debugging tiles with no data:
+
+1.  Confirm that the corresponding chart has data in Grafana as well. If your chart does not have data in Grafana, the chart in the converted dashboard most likely will not have data either.
+1.  Confirm all templating variables referenced in the queries exist in the Cloud Monitoring
+    dashboard. There are some templating variables that the importer doesn't handle. See the [missing templating variable section](#templating-variables-are-missing)
+1.  Confirm the metric(s) being referenced exist in Cloud Monitoring and are not
+    inactive. You can do so by searching for the query's metric in the
+    [Metrics Explorer](https://console.cloud.google.com/monitoring/metrics-explorer). If the metric doesn't exist, you will need to find an alternative metric.
+1.  Confirm all the fields that are being filtered on in the query also exist in Cloud Monitoring.
```

Comment:
A bit confusingly written - I'd say to first look at the query to determine if there are any hard equality matchers, then go to ME and look it up and ensure the label key and value both exist.

Reply:
I've updated the wording to use matchers. I've also updated the instructions here to align with what you suggested.